pulsar-client/src/main/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImpl.java

```diff
@@ -920,7 +919,6 @@ private void handleSubscribeOneTopicError(String topicName, Throwable error, Com
                         log.warn("[{}] Failed to subscribe for topic [{}] in topics consumer, subscribe error: {}",
                             topic, topicName, error.getMessage());
                         topics.remove(topicName);
-                        checkState(allTopicPartitionsNumber.get() == consumers.values().size());
```

Comment:
This is one topic subscribe error, so we should remove the check with all topic partitions number, Otherwise, the one topic subscribe future can't complete.